Formula/umple.rb

```diff
@@ -1,19 +1,16 @@
 class Umple < Formula
-  desc "Umple: Model-Oriented Programming - embed models in code and vice versa and generate complete systems"
+  desc "Modeling tool/programming language that enables Model-Oriented Programming"
   homepage "https://www.umple.org"
   url "https://github.com/umple/umple/archive/refs/tags/v1.30.2.tar.gz"
   sha256 "fc2a880c876fdebc00657e2d4461ad5237bd85bda85742778b2ec49c143c31d4"
   license "MIT"
 
-  bottle :unneeded
-
-  depends_on "ant" => "with-ivy"
+  depends_on "ant"
   depends_on "ant-contrib"
   def install
-    # Use IO.popen instead of system if user should see progress of install so it doesn't look like it is hanging
-    system "export UMPLEROOT=$(pwd); dev-tools/fbumple"
-    # IO.popen("export UMPLEROOT=$(pwd); dev-tools/fbumple") { |io| while (line = io.gets) do puts line end }
-    system "cd dist; echo '#!/bin/csh -f\njava -jar #{prefix}/libexec/'$(ls umple-*.jar)' $*' > ../umple"
+    ENV["UMPLEROOT"] = buildpath
+    system "dev-tools/fbumple"
+    system "cd dist; echo '#!/bin/csh -f\njava -jar #{libexec}/'$(ls umple-*.jar)' $*' > ../umple"
```

Comment:
Please use `write_jar_script`.

Reply:
Hi @carlocab , will replacing system with bin.write_jar_script do the trick? I'm not familiar with write_jar_script.

Reply:
It should produce a similar script, yes. You can `grep` the repo for examples of its usage; there are plenty. There's also documentation on it: https://rubydoc.brew.sh/Pathname.html#write_jar_script-instance_method